package.json

```diff
@@ -51,7 +51,8 @@
     "lint": "nx workspace-lint && ng lint",
     "dep-graph": "nx dep-graph",
     "help": "nx help",
-    "workspace-generator": "nx workspace-generator"
+    "workspace-generator": "nx workspace-generator",
+    "eslint-plugin:update": "ts-node ./build/generate-eslint-plugin.ts"
```

Comment:
This command will:

- update the rules docs page (e.g. updates description or config)
- the overview page (e.g. adds a new rule to the page)
- generate the ESLint configs based on the configuration of the rules


This command was a pre-commit hook in the ngrx-eslint-plugin, but I don't know how we want to use it within the NgRx repo. For now, it's a manual command, which is fine imo.

Reply:
Is there a way to only run it for changes to the walking plugin folders?

Reply:
It should be possible with a custom hook I think, I'll verify this later.